recipe/Makefile.vc

```diff
@@ -0,0 +1,257 @@
+# To build locally get a copy of the dependancies (http://windows.php.net/downloads/php-sdk/deps/)
```

Comment:
is it possible to do this as a patch on the source version? or just too different?

Reply:
I think the problem is more that it's using a non-autogen Makefile.vc here and I'm not skilled enough to write an autoconf for Windows either.

Reply:
well, it would be cMake if we wanted to try to auto-generate this.

I working on the Mac build now -- maybe I'll try cmake there.